macros/src/command/slash.rs

```diff
@@ -215,7 +215,7 @@ pub fn generate_context_menu_action(
     Ok(quote::quote! {
         <#param_type as ::poise::ContextMenuParameter<_, _>>::to_action(|ctx, value| {
             Box::pin(async move {
-                if !ctx.framework.options.manual_cooldowns {
+                if !ctx.framework.options.manual_cooldowns && !ctx.command.manual_cooldowns {
```

Comment:
(nitpick, you may ignore to your own discretion) There's three of these `!ctx.framework.options.manual_cooldowns && !ctx.command.manual_cooldowns` now, maybe they can be centralized somewhere. So that it's harder to forget one of the two clauses in future code, if another one of these if's will be added in the future

Reply:
Would involve making a function public somewhere that would then allow users to interact with, not sure if that is favorable, the logic itself isn't that complex so I don't believe its necessary at this time.